source/Pacioli/Pacioli.WebApi/Controllers/UserController.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.IdentityModel.Tokens;
+using Pacioli.Lib.Models;
+using Pacioli.WebApi.Models;
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Pacioli.WebApi.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UserController : ControllerBase
+    {
+        private readonly UserManager<Accountant> _userManager;
```

Comment:
Since this is a generic User controller, perhaps we should inject a generic user manager? The accountant role can be picked up by 
```CSharp
var userRoles = await _userManager.GetRolesAsync(user);
```